papaparse.js

```diff
@@ -415,6 +415,50 @@
 		}
 	}
 
+	function DynamicTypingConfig(config)
+	{
+		this._columnConfig = {};
+		this._isAllDynamic = false;
```

Comment:
Not necessarily, if `dynamicTyping` was set to `true`, then it will be marked as `true`. There are tests for that already. See line 424.